onboarding/src/Components/FormsEdit/FormsSectionsPreview.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import parse from "html-react-parser";
+
+function FormsSectionsPreview({ section }) {
+  const answersList = section.data.map((answer, i) => {
+    if (section.type === "osa" || section.type === "msa") {
+      return (
+        <tr key={ `answ-${answer.id}` }>
+          <td className="d-flex justify-content-between align-items-center pr-0">
+            <div className="w-100">
+                <div
+                  className={`custom-control custom-control-inline custom-${
+                    section.type === "osa" ? "radio" : "checkbox"
+                  }`}
+                >
+                  <input
+                    id={ `answ-${answer.id}` }
+                    className="custom-control-input"
+                    type={ section.type === "osa" ? "radio" : "checkbox" }
+                  />
```

Comment:
`input`s are not checked when user did that before.
Is it expected result?
... or it doesn't matter at all :)

Reply:
In my opinion preview is to see, what the employee will see, that's why answers are not checked in preview.